activerecord/test/schema/schema.rb

```diff
@@ -562,6 +562,13 @@
     t.timestamps null: true
   end
 
+  disable_referential_integrity do
```

Comment:
Why use `disable_referential_integrity` here?

Reply:
Thanks for the reminder, I was getting FK errors during development but they're fine now. Fixed this + the alphabetical order.